recipes/instaseis/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "7fece415cca6c70e575531a463c0d5cb6d30f59740d7a113a02d584775a8b0c1" %}
+
+package:
+  name: instaseis
+  version: {{ version }}
+
+
+source:
+  fn: instaseis-{{ version }}.zip
+  url: https://pypi.io/packages/source/i/instaseis/instaseis-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+  run:
+    - python
+    - libgfortran
+    - numpy
+    - h5py
+    - obspy
+    - future
+    - tornado
+    - requests
+    - jsonschema
+    - pyqt
+    - pyqtgraph
+
+test:
+  imports:
+    - instaseis
+  commands:
```

Comment:
I was a bit cautious about this because it might just set a very low ulimit for somebodies machine if they choose to run this which might immediately crash the machine. I think the current solution is safe enough.

Is there a way to have selectors in the `meta.yaml` file check for the existence of environment variables? That would make it straightforward to check if something runs in travis or not.